sunpy/net/dataretriever/tests/test_kanzelhohe.py

```diff
@@ -0,0 +1,60 @@
+#This module was developed with funding provided by
+#the Google Summer of Code 2016.
+import datetime
+import pytest
+
+from sunpy.time.timerange import TimeRange
+from sunpy.net.vso.attrs import Time,Instrument,Level
+from sunpy.net.dataretriever.client import QueryResponse
+from sunpy.net.dataretriever.downloader_factory import UnifiedResponse
+from sunpy.net import Fido
+from sunpy.net import attrs as a
+
+import sunpy.net.dataretriever.sources.kanzelhohe as kanzelhohe
+
+LCClient = kanzelhohe.KanzelhoheClient()
```

Comment:
LC comes from ligthcurve, these are images, so I would suggest to change the "client" name to KClient or something more meaningful in this context.